src/Brouter/Bit.Brouter.Demo/Client/Pages/McpPage.razor.css

```
@@ -0,0 +1,104 @@
+    color: var(--bb-text-muted);
+    font-size: .82rem;
+}
+
+/* The outputs are long by nature - a tool answer is a document. They scroll in place rather than
+   pushing the next section a screen and a half down. */
+.mcp-output,
+.mcp-wire {
+    max-height: 22rem;
+    overflow: auto;
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)